src/i18n/locales/en/translation.json

```diff
@@ -58,20 +53,14 @@
   "wallet_address": "Your wallet address:",
   "wallet_connect_failed": "<0><p>Failed to connect to wallet. Please reload the page and try again.</p> <1>Help with Brave Wallet</1></0>",
   "wallet_connect_request": "Requesting wallet access...",
-  "web3_cta_fine_print": "Learn more about Web3 Talk calls.",
-  "web3_premium_trial": "<0><p>Brave Talk premium subscription required for Web3 calls. Already have premium? <1>Log in</1></p></0>",
-  "web3_sign_up_button": "Sign up to host a Web3 Call",
   "welcome_page_button_download_text": "Download Brave",
   "welcome_page_button_hollow": "Start free trial",
   "your_avatar_nft_header": "Your Avatar NFT",
   "premium_card_title": "Premium calls",
-  "web3_card_title": "Host a Web3 call",
-  "web3_account_body": "Choose the network of the account you want to connect start the Web3 call.",
   "leo_promo_headline": "Introducing Leo AI for meetings",
   "leo_promo_description": "Never take meeting notes again. Leo brings AI to your meetings with AI summaries. Get transcripts and use AI chat to get answers from your meetings, and more.",
   "leo_promo_link_text": "Learn more.",
-  "web3_sunset_title": "ATTENTION",
-  "web3_sunset_description": "Web3 call access is being disabled. On or about March 17, 2025, the \"Host a Web3 Call\" button will be removed from Brave Talk, and you will no longer be able to host Web3 calls.",
+  "web3_calls_no_longer_supported": "Web3 calls are no longer supported. Please create a new room.",
```

Comment:
do we have a plan to use this translation, or should we just rely on the server-defined English error message? I recommend the latter